dailystats.py

```diff
@@ -8,14 +8,14 @@
 from datetime import datetime
 
 
-debug = False
-input_file_name = "C:\\Geraldine\\Coronavirus\\coronavirus-cases20200427.csv"
-output_file_name = "C:\\Geraldine\\Coronavirus\\coronavirus-output_results.csv"
-start_date_string = "2020-03-01"
-peak_date_string = "2020-04-08"
+DEBUG = False
+INPUT_FILE_NAME = "C:\\Geraldine\\Coronavirus\\coronavirus-cases20200427.csv"
+OUTPUT_FILE_NAME = "C:\\Geraldine\\Coronavirus\\coronavirus-output_results.csv"
+START_DATE_STRING = "2020-03-01"
+PEAK_DATE_STRING = "2020-04-08"
 
-date_object = datetime.strptime(start_date_string, "%Y-%m-%d")
-peak_date_object = datetime.strptime(peak_date_string, "%Y-%m-%d")
+DATE_OBJECT = datetime.strptime(START_DATE_STRING, "%Y-%m-%d")
+PEAK_DATE_OBJECT = datetime.strptime(PEAK_DATE_STRING, "%Y-%m-%d")

```

Comment:
Python doesn't have concepts like `static` or `final` - but it does have naming conventions for communicating when something should be constant. Module-level constants are all caps (like C #define)